commodore/package/compile.py

```diff
@@ -36,13 +35,13 @@ def compile_package(
     # relative includes internally, the package name shouldn't matter for compilation.
     pkg_name = pkg_path.stem.replace("package-", "")
 
-    # Verify root class
-    root_class_path = root_class.replace(".", "/")
-    root_cls = pkg_path / f"{root_class_path}.yml"
-    if not root_cls.is_file():
-        raise click.ClickException(f"Root class '{root_cls.stem}' doesn't exist.")
+    root_class_name = root_class.replace(".yml", "").replace("/", ".")
+    # Convert root class file to class name
+    root_class_path = pkg_path / root_class
+    if not root_class_path.is_file():
+        raise click.ClickException(f"Root class '{root_class_name}' doesn't exist.")
```

Comment:
The naming is a bit inconsistent here. You call it test class in the documentation but root class in the error.

Reply:
I missed renaming that one... fixed now.